xapian-letor/scorer/ndcg_score.cc

```diff
@@ -0,0 +1,71 @@
+/** @file ndcg_score.cc
+ *  @brief Implementation of NDCGScore
+ */
+/* Copyright (C) 2016 Ayush Tomar
```

Comment:
Sorry, by fresh I was just confirming that you wrote it yourself. I wasn't expecting it to be a novel twist on NDCG itself. It may be worth pointing to a good public definition of NDCG though, now I think about it.